pkg/disk/bdf.go

```diff
@@ -476,3 +476,105 @@ func checkVfhpOnline() {
 func IsVFInstance() bool {
 	return isVFInstance
 }
+
+type MachineType int
+
+const (
+	BDF   MachineType = iota // 0
```

Comment:
Should we add an Unknown MachineType as the default value, to avoid uninitialized variable being interpreted as BDF?

Reply:
I believe this is unnecessary since all the initialization code is under control.

Reply:
OK, as long as we don't have something like ParseMachineType(). Can we replace `VMOC_MODE` env with something like `RUND_MACHINE_TYPE`, to prepare for even more types?

Reply:
sounds good